jormungandr-integration-tests/src/common/jormungandr/starter.rs

```diff
@@ -4,14 +4,21 @@ use self::custom_error::custom_error;
 use crate::common::configuration::jormungandr_config::JormungandrConfig;
 use crate::common::file_utils;
 use crate::common::jcli_wrapper;
-use crate::common::jormungandr::{commands, process::JormungandrProcess};
+use crate::common::jormungandr::{
+    commands, logger::JormungandrLogger, process::JormungandrProcess,
+};
 
 use crate::common::process_assert;
 use crate::common::process_utils::{self, output_extensions::ProcessOutput, ProcessError};
-use std::process::{Child, Command, Output};
-
+use std::{
+    path::PathBuf,
+    process::{Child, Command, Output},
+    time::{Duration, Instant},
+};
 custom_error! {pub StartupError
```

Comment:
I advise converting uses of `custom_error` to [thiserror](https://crates.io/crates/thiserror) when you have an opportunity. Just as easy to use, but without the ugly custom syntax and you can explicitly control which `From` impls get generated.

Reply:
I'll implement it in separate PR